cmd/pbstatuser/main.go

```diff
@@ -112,6 +126,7 @@ func checkSourceAvailabilityAzure(ctx context.Context) {
 		// TODO: check if source is avavliable - WIP
 		sr.Status = kafka.StatusAvaliable
 		chSend <- sr
+		totalAvailabilityCheckReqs.WithLabelValues("azure", string(sr.Status), "").Inc()
```

Comment:
The newly created package could expose functions which are typed and actually hide the implementation which is Prometheus today: `func IncTotalAvailablilityCheckReqs(string, StatusType, error)` for example. It would have been cleaner than exposing bare bone Prometheus metric where the caller actually must know all the labels.

Reply:
great idea, thank you!